src/tests/fixtures/projectVotes.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
+        "id": "0x9581ffb8b5f50015331432891cec32309c1366360ecbfa9d0b8336302b11811a",
+        "projectId": "0xA0000000000000000000000000000000000000000000000000000000000001",
+        "roundId": "0xA000000000000000000000000000000000000000",
+        "token": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+        "voter": "0x7A45E23cbb7765900f02c54757498beF01C2b18F",
+        "grantAddress": "0x4557B18E779944BFE9d78A672452331C186a9f48",
```

Comment:
what do you think if for tests we usefake addresses like `0x0000000000000000000000000000000000000000`, `0x0000000000000000000000000000000000000001`, etc?
I don't see any problem in using real addresses but it only feels strange if we can search for them on etherscan and they are actually used by real ENS names :) wdyt?